src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -246,6 +246,10 @@ std::vector<std::string> disabledTestPatterns() {
         retVector.emplace_back(R"(.*Snippets.*MHA.*)");
         retVector.emplace_back(R"(.*Snippets.*(MatMul|Matmul).*)");
     }
+    if (!InferenceEngine::with_cpu_x86_fp16()) {
+        // Skip fp16 tests for paltforms that don't support fp16 precision
+        retVector.emplace_back(R"(.*INFERENCE_PRECISION_HINT=(F|f)16.*)");
+    }
```

Comment:
You add with_cpu_x86_fp16() to help tell capability of avx512_bf16 and use it.
https://github.com/openvinotoolkit/openvino/pull/18227/files#diff-2ec2dbc37b12dc344e3d56d476bebb0a795777886906b5598ec2c91873404c0dR76
1. There is no limitation to support f16 only on avx512_bf16. If on avx2, can support it directly. If no avx2, should be supported with fallback on f32 internally.
2. The function name is ambiguous, maybe we can remove it as no needed.

Reply:
Agreed and applied. Thanks Chenhu! And to make this happen, this config [here](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/config.cpp#L182) is also revised to be avx2 for fp16. Please feel free to commet, if there should be any concerns.
cc @dmitry-gorokhov @usstq

Reply:
For the change in common part, should we have performance test? As f16 is not always outperform f32, reduce maybe ok as the computing complexity is low, but not sure for other computing bound node and shapes as precision convert overhead introduced. Given not all nodes support f16, so f16 hint is actually mixed precision in execution.

Reply:
According to our discussion, for now we only activate FP16 test cases for platforms with tAVX512_FP16 instructions. FP16 test cases for AVX2 will be activated after the follow-ups in https://github.com/openvinotoolkit/openvino/pull/16500#discussion_r1236404957 be applied. Thanks Chenhu!